<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindMusicWidget</class>
 <widget class="QWidget" name="FindMusicWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,1">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1,0,1">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="searchBox">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
        border-radius: 4px;
        height: 25px;
        border: 1px solid rgb(100, 100, 100);
        background: rgb(72, 72, 73);
}
QLineEdit:enabled {
        color: rgb(175, 175, 175);
}
QLineEdit:enabled:hover, QLineEdit:enabled:focus {
        color: rgb(230, 230, 230);
}
QLineEdit:!enabled {
        color: rgb(155, 155, 155);
}</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSearch">
       <property name="styleSheet">
        <string notr="true">QPushButton:enabled {
	border:none;
        background: rgb(81, 81, 81);
        	color: rgb(188, 188, 188);
}
QPushButton:!enabled {
        background: rgb(100, 100, 100);
        color: rgb(200, 200, 200);
}
QPushButton:enabled:hover{
        background: rgb(85, 85, 85);
}
QPushButton:enabled:pressed{
        background: rgb(80, 80, 80);
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../mainWindow/mainwindow.qrc">
         <normaloff>:/image/resource/search.png</normaloff>:/image/resource/search.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="styleSheet">
      <string notr="true">QTabWidget:pane {
    border: none;
}

QTabBar:tab {
    background: rgba(0, 0, 0, 120);
    color: #a59400;
    width: 128px;
    height: 19px;
    border-top-left-radius: 3px;
    border-top-right-radius: 3px;
    margin-right: 2px;
}

QTabBar:tab:selected {
    background: rgba(0, 0, 0, 166);
    color: #fae119;
}

QTabBar:tab:hover:!selected {
    background: rgba(0, 0, 0, 156);
}</string>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="tabMusic">
      <attribute name="title">
       <string>音乐</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="MusicTableWidget" name="musicTable">
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <column>
          <property name="text">
           <string>音乐标题</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>歌手</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>时长</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabAlbum">
      <attribute name="title">
       <string>专辑</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="StyledTableWidget" name="albumTable">
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>300</number>
         </attribute>
         <column>
          <property name="text">
           <string>专辑名称</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>歌手</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabMusiclist">
      <attribute name="title">
       <string>歌单</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="StyledTableWidget" name="listTable">
         <property name="styleSheet">
          <string notr="true">QTabWidget:pane {
    border: none;
}

QTabBar:tab {
    background: rgba(0, 0, 0, 120);
    color: #a59400;
    width: 128px;
    height: 19px;
    border-top-left-radius: 3px;
    border-top-right-radius: 3px;
    margin-right: 2px;
}

QTabBar:tab:selected {
    background: rgba(0, 0, 0, 166);
    color: #fae119;
}

QTabBar:tab:hover:!selected {
    background: rgba(0, 0, 0, 156);
}</string>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <column>
          <property name="text">
           <string>音乐列表</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>歌曲数</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>创建用户</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>播放</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>收藏</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MusicTableWidget</class>
   <extends>QTableWidget</extends>
   <header location="global">musictablewidget.h</header>
  </customwidget>
  <customwidget>
   <class>StyledTableWidget</class>
   <extends>QTableWidget</extends>
   <header location="global">styledtablewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../mainWindow/mainwindow.qrc"/>
 </resources>
 <connections/>
</ui>
